src/components/Book.js

```diff
@@ -0,0 +1,28 @@
+const Book = (book) => {
+  const { title, author } = book('');
+  return (
```

Comment:
- [ ] There is a small syntax error here. Props should not contain parentheses, kindly fix it 👍 

It should be 💡
```js
  const { title, author } = book;
```

![image](https://user-images.githubusercontent.com/45231276/208221603-30e10adc-b9e4-4c34-b5ed-5cac857cd4ee.png)